util/plugins/eu.esdihumboldt.util.resource.schemas.end/plugin.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="eu.esdihumboldt.hale.ui.schema.presets">
+      <category
+            id="eu.esdihumboldt.util.resource.schemas.end.category"
+            name="European Noise Directive">
+      </category>
+      <schema
+            category="eu.esdihumboldt.util.resource.schemas.end.category"
+            description="Geopackage European Noise Directive Application Schema for DF1_5 Agglomeration Source"
+            id="eu.esdihumboldt.util.resource.schemas.presets.end.df1-5.agg"
+            name="END DF1_5 Agglomeration Source"
+            version="1.0">
+         <uri
+               value="https://raw.githubusercontent.com/wetransform-os/geopackage-end/018d8207556dac0f0e4977cbe2a2206cf9094d9c/DF1_5/AgglomerationSource/AgglomerationSource.hsd.json">
```

Comment:
So that commit basically represents version 1.0?

Reply:
No, the commit id is the latest commit id. As @thorsten-reitz mentioned in the ticket [ING-2970](https://wetransform.atlassian.net/browse/ING-2970) to tag these commits as `1.0` and if I add `1.0` in the `tag` element then in the application it would be shown as `(1.0)` and will be inconsistent with the other versions that are shown. So I decided to add `1.0` in the version field. I believe it is ok.